tests/e2e/cairo1/test_cairo1_building.py

```diff
@@ -9,14 +10,37 @@
 def test_cairo1_build(protostar: ProtostarFixture, copy_fixture: CopyFixture):
     copy_fixture("cairo1_build", "./cairo_project")
     os.chdir("./cairo_project")
-    protostar(["build"])
+
+    result = protostar(["build"])
+
+    assert 'Class hash for contract "main": 0x' in result
+    assert 'Compiled class hash for contract "main": 0x' in result
+
     compiled_sierra_path = Path("build/main.sierra.json")
     compiled_casm_path = Path("build/main.casm.json")
     assert compiled_sierra_path.exists()
     assert compiled_sierra_path.read_text()
     assert compiled_casm_path.exists()
     assert compiled_casm_path.read_text()
 
+    class_hash_path = Path("build/main.class.hash")
+    compiled_class_hash_path = Path("build/main.compiled.class.hash")
```

Comment:
I'm not sure where exactly this change was introduced, but I don't think we should be using `compiled.class.hash`.

If we want to have a custom extension for these I'd change it to `.compiled_class_hash` and `.compiled_class` respectively.